dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/configuration/CorsConfig.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dolphinscheduler.api.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.cors.CorsConfiguration;
+import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
+import org.springframework.web.filter.CorsFilter;
+
+/**
+ * core filter
+ */
+
+@Configuration
+public class CorsConfig {
+
+    @Bean
+    public CorsFilter corsFilter() {
+        CorsConfiguration config = new CorsConfiguration();
```

Comment:
You don't need to create this `CorsConfig.java` file, and just append `.allowCredentials(true)`
in `addCorsMappings ` like:
```
registry.addMapping(PATH_PATTERN).allowedOrigins("*").allowedMethods("*").allowedHeaders("*").allowCredentials(true);
```
At the same time, the axios in front-end also needs to add `withCredentials: 
 true`, otherwise the error will still occur under cors.

Reply:
I tried to modify it this way, but I still get an error in the two situations written in my issue

Reply:
Do you moidfy front-end?

Reply:
![image](https://user-images.githubusercontent.com/20266837/104561108-d3da4880-5681-11eb-908b-6696397b4aaf.png)

Reply:
Let me check it.